example/spectrogram/app.js

```diff
@@ -51,4 +51,13 @@ document.addEventListener('DOMContentLoaded', function() {
     })();
 
     wavesurfer.load('../media/demo.wav');
+}
+
+// Init & load
+document.addEventListener('DOMContentLoaded', function() {
+    WaveSurfer.util
```

Comment:
add comment what's being loaded

Reply:
I added a comment.  Not sure if that was what you were looking for.